client/src/app/core/core-services/http.service.ts

```diff
@@ -273,4 +273,27 @@ export class HttpService {
     public async delete<T>(path: string, data?: any, queryParams?: QueryParams, header?: HttpHeaders): Promise<T> {
         return await this.send<T>(path, HTTPMethod.DELETE, data, queryParams, header);
     }
+
+    /**
+     * Retrieves a binary file from the url and returns a base64 value
+     *
+     * @param url file url
+     * @returns a promise with a base64 string
+     */
+    public async convertUrlToBase64(url: string): Promise<string> {
```

Comment:
The name is wrong. The url is not converted to b64, but the content when downloaded. Maybe this can be changed